lib/fprotect/fprotect_acl.c

```diff
@@ -7,6 +7,12 @@
 #include <hal/nrf_acl.h>
 #include <errno.h>
 
+#ifdef CONFIG_SOC_NRF5340_CPUNET
+#define CODEPAGESIZE NRF_FICR->INFO.CODEPAGESIZE
```

Comment:
Why not include `<hal/nrf_ficr.h>` and use `nrf_ficr_codesize_get(NRF_FICR)` instead?

Reply:
updated